docs/install_guides/installing-openvino-apt.md

```diff
@@ -15,150 +15,119 @@ The complete list of supported hardware is available in the [Release Notes](http
 - Ubuntu 18.04.x long-term support (LTS), 64-bit
 - Ubuntu 20.04.0 long-term support (LTS), 64-bit
```

Comment:
@xu-yuan1 , @Jacob27
I suggest to use <major>.<mibor> version for specifying Ubuntu version in a system requirements:

- Ubuntu 18.04 long-term support (LTS), 64-bit
- Ubuntu 20.04 long-term support (LTS), 64-bit

Reply:
Updated. I will check elsewhere to align to this format.

Reply:
no, I believe it is a misprint.

Reply:
BTW, `macOS* 10.15.x` should be also corrected to `macOS* 10.15` .

Reply:
Thanks! All updated. :)